### DISCUSSION TITLE
In the name of the Lord - please fix VPN connectivity con WSL2

### DISCUSSION THREAD

**Discussion by willkaman** [microsoft/WSL#9494] [General]
None of the workarrounds has worked for me. in not gonna waste any more minute trying to fix this BASIC feature.
I'm in love with WSL, but this is just killing me.

**Comment by phil-blain** [comment]
Hi !  Did you try https://github.com/sakai135/wsl-vpnkit/ ? 

It fixes network connectivity for me (with Cisco Annyconnect).
